src/NzbDrone.Core/Notifications/Plex/Server/PlexServerSettings.cs

```diff
@@ -43,6 +45,15 @@ public PlexServerSettings()
         [FieldDefinition(5, Label = "Update Library", Type = FieldType.Checkbox)]
         public bool UpdateLibrary { get; set; }
 
+        [FieldDefinition(6, Label = "Use Mapping", Type = FieldType.Checkbox)]
+        public bool UseMapping { get; set; }
+
+        [FieldDefinition(7, Label = "Map From", Type = FieldType.Textbox)]
```

Comment:
This should be an advanced setting as it is not relevant to all OS

Reply:
translates as well, no?

Reply:
> translates as well, no?

The whole notification section needs translating at the moment and extending to support translates; it's on my todo list which is every growing at the moment!